scripts/communication.js

```diff
@@ -4,6 +4,10 @@
    If somebody in chat write appropriate query, mona will answer him a random response for this query.
 */
 
+var englishGreetings = ['Hello', 'Good morning', 'Hi, team!'],
```

Comment:
@roman-dubrovsky can you put the array declaration under require directive?
Everything except that is good to merge as for me.

Reply:
@Nailan fixed. So can you merge and deploy this PR?

Reply:
@roman-dubrovsky done